apps/comps/app/layout.tsx

```diff
@@ -20,11 +21,12 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en" suppressHydrationWarning className="overflow-x-hidden">
+    <html lang="en" suppressHydrationWarning>
```

Comment:
@dtbuchholz I reverted the change that was meant to fix an issue on the landing page because it was causing a side effect when popovers were rendered. Since the landing page wasn’t fully functional and isn’t part of Jovi’s new designs, I’ve set it to redirect to /competitions, which is supposed to be the landing page for now